notebooks/official/matching_engine/sdk_matching_engine_create_stack_overflow_embeddings.ipynb

```diff
@@ -0,0 +1,1131 @@
+{
```

Comment:
<p>same comment about bold text in the second sentence </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1575/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>You mean, unbold it?</p>